tests/h/tasks/indexer_test.py

```diff
@@ -79,9 +80,23 @@ def search_index(self, search_index):
 
 
 class TestReportJobQueueMetrics:
-    def test_it(self):
+    def test_it(self, factories, newrelic, search_index):
+        factories.Job.create()
+        factories.Job.create(expires_at=datetime.utcnow() - timedelta(minutes=1))
+
         indexer.report_job_queue_metrics()
 
+        newrelic.agent.record_custom_metrics.assert_called_once_with(
+            [
+                ("Custom/Job/Queue/Length", 2),
+                ("Custom/Job/Queue/Expired/Length", 1),
+                (
+                    "Custom/SyncAnnotations/Queue/API/Length",
+                    search_index._queue.count.return_value,
+                ),
+            ]
+        )
```

Comment:
Since this method now actually does something (it actually counts different types of job in the DB) it's now worth adding an assert here